app/assets/v2/js/board.js

```diff
@@ -318,20 +322,34 @@ Vue.mixin({
     async deployAndSaveToken() {
       const vm = this;
 
+      if (!web3) {
+        _alert('Please connect a wallet', 'error');
+      }
       [user] = await web3.eth.getAccounts();
-      // TODO: This is a deterministic localhost address. Should be an env variable for rinkeby/mainnet.
-      const factoryAddress = '0x7bE324A085389c82202BEb90D979d097C5b3f2E8';
-      const mockDaiAddress = '0x6b67DD1542ef11153141037734D21E7Cbd7D9817';
+
+      // We currently have DAI addresses hardcoded, so right now pTokens only support
+      // being priced in DAI
+      let tokenAddress;
```

Comment:
I feel like simply calling this `tokenAddress` may be confused with `pTokenAddress`. At risk of being overly explicit, perhaps we call this `purchaseTokenAddress`?

Reply:
Updated in b4ec2ce